<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDSPRxDNBDialog</class>
 <widget class="QDialog" name="WDSPRxDNBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise Blanker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="activeLayout">
     <item>
      <widget class="QLabel" name="nbLabel">
       <property name="text">
        <string>Scheme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nb">
       <item>
        <property name="text">
         <string>NB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NB2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="nbLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nb2ModeLabel">
       <property name="text">
        <string>NB2 mode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="nb2Mode">
       <property name="toolTip">
        <string>Noise blanking #2 mode</string>
       </property>
       <item>
        <property name="text">
         <string>Zero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sample&amp;Hold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mean Hold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hold Sample</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interpolate</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="nbLeadTimeLabel">
       <property name="text">
        <string>Lead time (ms)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nbSlewTimeLabel">
       <property name="text">
        <string>Slew time (ms)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="nbSlewTime">
       <property name="toolTip">
        <string>Noise blanking slew time (ms)</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nbLagTimeLabel">
       <property name="text">
        <string>Lag time (ms)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="nbThresholdLabel">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="nbLeadTime">
       <property name="toolTip">
        <string>Noise blanking lead time (ms)</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="nbThreshold">
       <property name="toolTip">
        <string>Noise blanking threshold multiplier</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="nbLagTime">
       <property name="toolTip">
        <string>Noise blanking lag time (ms)</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nbAvgTimeLabel">
       <property name="text">
        <string>Avg time (ms)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="nbAvgTime">
       <property name="toolTip">
        <string>Noise blanking averaging time (ms)</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WDSPRxDNBDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WDSPRxDNBDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
